src/vellum/workflows/runner/runner.py

```diff
@@ -373,6 +373,12 @@ def initiate_node_streaming_output(output: BaseOutput) -> None:
         except Exception as e:
             logger.exception(f"An unexpected error occurred while running node {node.__class__.__name__}")
 
+            node_module = node.__class__.__module__
```

Comment:
we actually need to check the module of where the exception came from. Is that possible?